### PR TITLE
Live page docs update

### PR DESCRIPTION
**Pull request by ChristopherGS**
Please merge after the new SQL editor is turned on by default @petyosi

### CODE REVIEW

File: docs/guides/web-ui/live.md [hunk 1]
```diff
@@ -47,72 +54,56 @@ CREATE TABLE records AS (
...
+an LLM running with [PydanticAI](https://github.com/pydantic/pydantic-ai).
 
-5. **Link to the current view:** Clicking this copies a link to the page you are on, with the same query etc.
+### Common Clauses
+Common clauses: A list of pre-populated useful queries. A powerful way to rapidly generate the query you need, whilst
+simultaneously learning more about all the ways you can search your data. Clicking multiple clauses will add them to
+your query with an `AND` statement. If you'd like something other than an `AND` statement you can either modify the
+SQL populated in the SQL search or use the [explore screen](explore.md) for more complex queries.
 
-6. **Organization selection panel:** Opens a drawer with links to the different organizations you are a member of, and also has links to the Terms and Conditions, Support, Documentation, and a Log Out button.
```
Comment (ChristopherGS) [inline docs/guides/web-ui/live.md:72]: I've deliberately culled a lot of these - some are no longer relevant, but the section is currently just a bit too much of a "wall of text". I've tried to pull out the key/least intuitive areas to describe

File: docs/guides/web-ui/live.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
 # Live View
 
-The live view is the main view of **Logfire**, where you can see traces in real-time.
+The live view is a central part of **Logfire**, where you can see traces in real-time.
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
The live view is the focal point of **Logfire**, where you can see traces arrive in real-time.
```

File: docs/guides/web-ui/live.md [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
-The live view is useful (as the name suggests) for watching what's going on within your
-application in real-time, but it can also be used to explore historical data.
+The live view is useful for watching what's going on within your
+application in real-time (as the name suggests), but it can also be used to explore historical data via the *live view search*.
 
-## The Live View SQL Box
+## The Live View Search
 
-The live view has a query box at the top. Here you can enter the `WHERE` clause of a SQL query.
+To view the search box, click on `Search your spans` (keyboard shortcut `/`), this
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
To search the live view, click `Search your spans` (keyboard shortcut `/`), this
```

File: docs/guides/web-ui/live.md [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
+will open the search box:
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion 
opens the search pane:
```

File: docs/guides/web-ui/live.md [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
-The live view is the main view of **Logfire**, where you can see traces in real-time.
+The live view is a central part of **Logfire**, where you can see traces in real-time.
 
-The live view is useful (as the name suggests) for watching what's going on within your
-application in real-time, but it can also be used to explore historical data.
+The live view is useful for watching what's going on within your
+application in real-time (as the name suggests), but it can also be used to explore historical data via the *live view search*.
 
-## The Live View SQL Box
+## The Live View Search
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
## SQL search pane
```

File: docs/guides/web-ui/live.md [hunk 6]
```diff
@@ -1,15 +1,22 @@
 # Live View
 
-The live view is the main view of **Logfire**, where you can see traces in real-time.
+The live view is a central part of **Logfire**, where you can see traces in real-time.
 
-The live view is useful (as the name suggests) for watching what's going on within your
-application in real-time, but it can also be used to explore historical data.
+The live view is useful for watching what's going on within your
+application in real-time (as the name suggests), but it can also be used to explore historical data via the *live view search*.
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
application in real-time (as the name suggests). You can also explore historical data in the **search pane**.
```

**Review by summerscope — CHANGES_REQUESTED** [review]
@ChristopherGS I'm going to finish my review and instead just go ahead and commit my changes to your PR to speed this up 👍

File: docs/guides/web-ui/live.md [hunk 7]
```diff
@@ -1,15 +1,22 @@
...
+opens the search pane:
 
-![Logfire Live View SQL query box](../../images/guide/live-view-sql-box.png)
+![Search box](../../images/guide/live-view-search.png)
+
+### SQL Search
+Here you can write raw SQL. A good choice if you know what you want and/or love SQL. As the greyed out
+`SELECT * FROM RECORDS WHERE` implies, this is the `WHERE` clause of a SQL query.
+It has auto-complete & schema hints, so try typing something to get a reminder. To run your query click `Run` or keyboard
+shortcut `cmd+enter` (or `ctrl+enter` on windows/linux).
```
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
For confident SQL users, write your queries directly here. For devs who want a bit of help, try the new [PydanticAI](https://ai.pydantic.dev/) feature which generates a SQL query based on your prompt. You can also review the fields available and populate your SQL automatically using the `Reference` list, see more on this below.

**WHERE clause**
As the greyed out `SELECT * FROM RECORDS WHERE` implies, you're searching inside the `WHERE` clause of a SQL query. It has auto-complete & schema hints, so try typing something to get a reminder. To run your query click `Run` or keyboard shortcut `cmd+enter` (or `ctrl+enter` on windows/linux).
```

**Review by summerscope — APPROVED** [review]
ok @ChristopherGS please ignore my ugly commit history - made some edits to this copy to reflect latest changes, I think we will probably want to improve this more at some point but let's not let perfect be the enemy of better. FYI we should hold off on merging this PR until we make the new SQL editor the default mode - @petyosi could you merge this bad boy once that has happened?

**Pull request merged**